CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
   `policy_document_native_api true` is set. This new API is currently
   unreleased and unstable. It is recommended for testing and
   demonstration use only.
+* Update Chef to version 12.1.0
```

Comment:
Yeah, I was going to wait to merge this until then (and that'll be another PR against omnibus-chef)